rmf_traffic_ros2/src/rmf_traffic_ros2/schedule/Node.cpp

```diff
@@ -92,17 +92,22 @@ std::vector<ScheduleNode::ConflictSet> get_conflicts(
 
         const auto* dep_v =
           vc->route->check_dependencies(participant, plan_id, r);
-        if (dep_v && !dep_v->empty())
+        if (dep_v)
           continue;
 
         const auto* dep_u =
           route->check_dependencies(vc->participant, vc->plan_id, vc->route_id);
-        if (dep_u && !dep_u->empty())
+        if (dep_u)
           continue;
 
+//        std::cout << "No deps for " << description->name()
+//                  << " " << plan_id << "|" << r << " vs "
+//                  << vc->description.name() << " " << vc->plan_id << "|" << vc->route_id
+//                  << std::endl;
```

Comment:
Are these comments still useful?

Reply:
Thanks for catching that, comments are removed in [9433066](https://github.com/open-rmf/rmf_ros2/pull/205/commits/9433066fd92c2d53d5c3025e8bc88efe36f97bea)